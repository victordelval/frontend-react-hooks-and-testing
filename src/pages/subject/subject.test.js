import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Subject from "./subject";

configure({ adapter: new Adapter() });

describe("<Subject />", () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Subject />);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
