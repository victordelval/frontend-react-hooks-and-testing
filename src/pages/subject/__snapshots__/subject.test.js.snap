// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subject /> renders correctly 1`] = `ShallowWrapper {}`;
